components/movementsensor/adxl345/adxl345.go

```diff
@@ -294,7 +458,13 @@ func (adxl *adxl345) Accuracy(ctx context.Context, extra map[string]interface{})
 func (adxl *adxl345) Readings(ctx context.Context, extra map[string]interface{}) (map[string]interface{}, error) {
 	adxl.mu.Lock()
 	defer adxl.mu.Unlock()
-	return map[string]interface{}{"linear_acceleration": adxl.linearAcceleration}, adxl.err.Get()
+
+	readings := make(map[string]interface{})
+	readings["linear_acceleration"] = adxl.linearAcceleration
+	readings["single_tap_count"] = adxl.interruptsFound[SingleTap]
+	readings["freefall_count"] = adxl.interruptsFound[FreeFall]
```

Comment:
Does this count as a breaking change? Part of me says "yes, the readings used to have 1 key and now they have 3." Part of me says "no, the old behavior is unchanged and the new parts can be safely ignored by old code." What do other folks think?

Reply:
not a breaking change, you'd have to call the map key on the sensor readings to get to this data. I guess if someone was trying to access a specific index of the map that wasn't there, or that is now another key, this would be breaking. 

The sensor map can get very bloated very quickly. We should explore typing it more strongly for future components.